lib/parallel_report_portal/cucumber/report.rb

```diff
@@ -50,39 +50,44 @@ def launch_started(start_time)
           @launch_id
         end
       end
-      
+
       # Called to finish a launch. Any open children items will be closed in the process.
-      # 
+      #
       # @param clock [Integer] the millis from the epoch
       def launch_finished(clock)
         @tree.postordered_each do |node|
-          ParallelReportPortal.req_feature_finished(node.content, clock) unless node.is_root?
+          response = ParallelReportPortal.req_feature_finished(node.content, clock) unless node.is_root?
+          link = get_report_link(response)
+        end
+        response = ParallelReportPortal.req_launch_finished(launch_id, clock)
+        link = get_report_link(response)
+        if ParallelReportPortal.configuration.log_launch_link
+          print "Launch in ReportPortal: #{link}"
```

Comment:
This is a print and not a puts? Is the omission of a new line after this output deliberate?

Reply:
![image](https://user-images.githubusercontent.com/150938/209320896-8edf902a-b209-401f-85b4-604776503205.png)


Not deliberate, copy & pasted from the original PR - Seems happy as the print